bicep/main.bicep

```diff
@@ -0,0 +1,329 @@
+}
+
+//Find existing App Service Plan
+resource appServicePlan 'Microsoft.Web/serverfarms@2022-03-01' existing = {
+  name: appServicePlanName
+}
+
+//Create new App Service
+resource newAppService 'Microsoft.Web/sites@2022-03-01' = {
+  name: appServiceName
+  location: location
+  kind: 'app,linux'
+  properties: {
+    serverFarmId: appServicePlan.id
+    siteConfig: {
+      appSettings: [
+        {
+          name: 'POSTGRES_PASSWORD'
+          value: POSTGRES_PASSWORD
+        },{
+          name: 'POSTGRES_USER'
+          value: POSTGRES_USER
+        },{
+          name: 'RESET_SECRET'
+          value: RESET_SECRET
+        },{
+          name: 'SECRETS_121_SERVICE_SECRET'
+          value: SECRETS_121_SERVICE_SECRET
+        },{
+          name: 'USERCONFIG_121_SERVICE_PASSWORD_ADMIN'
+          value: USERCONFIG_121_SERVICE_PASSWORD_ADMIN
+        },{
+          name: 'ENV_ICON'
+          value: ENV_ICON
+        },{
+          name: 'APPLICATION_INSIGHT_ENDPOINT'
+          value: APPLICATION_INSIGHT_ENDPOINT
+        },{
+          name: 'APPLICATION_INSIGHT_IKEY'
+          value: appInsights.properties.InstrumentationKey
+        },{
+          name: 'APPLICATIONINSIGHTS_CONNECTION_STRING'
+          value: 'InstrumentationKey=${appInsights.properties.InstrumentationKey};IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/;ApplicationId=${appInsights.properties.AppId}'
+        },{
+          name: 'ApplicationInsightsAgent_EXTENSION_VERSION'
+          value: ApplicationInsightsAgent_EXTENSION_VERSION
+        },{
+          name: 'ENV_NAME'
+          value: ENV_NAME
+        },{
+          name: 'EXTERNAL_121_SERVICE_URL'
+          value: EXTERNAL_121_SERVICE_URL
+        },{
+          name: 'INTERSOLVE_PASSWORD'
+          value: INTERSOLVE_PASSWORD
+        },{
+          name: 'INTERSOLVE_USERNAME'
+          value: INTERSOLVE_USERNAME
+        },{
+          name: 'KOBO_CONNECT_API_URL'
+          value: KOBO_CONNECT_API_URL
+        },{
+          name: 'MOCK_COMMERCIAL_BANK_ETHIOPIA'
+          value: MOCK_COMMERCIAL_BANK_ETHIOPIA
+        },{
+          name: 'MOCK_INTERSOLVE'
+          value: MOCK_INTERSOLVE
+        },{
+          name: 'MOCK_SAFARICOM'
+          value: MOCK_SAFARICOM
+        },{
+          name: 'MOCK_SERVICE_URL'
+          value: MOCK_SERVICE_URL
+        },{
+          name: 'MOCK_TWILIO'
+          value: MOCK_TWILIO
+        },{
+          name: 'MOCK_VODACASH'
+          value: MOCK_VODACASH
+        },{
+          name: 'NODE_ENV'
+          value: NODE_ENV
+        },{
+          name: 'PORT_121_SERVICE'
+          value: PORT_121_SERVICE
+        },{
+          name: 'POSTGRES_DBNAME'
+          value: POSTGRES_DBNAME
+        },{
+          name: 'POSTGRES_HOST'
+          value: POSTGRES_HOST
+        },{
+          name: 'REDIRECT_PORTAL_URL_HOST'
+          value: REDIRECT_PORTAL_URL_HOST
```

Comment:
This value is also created by/in this script, so maybe it can be used from there as well?